lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -0,0 +1,336 @@
+const test = require('ava');
+const omit = require('lodash/omit');
+const cryptoRandomString = require('crypto-random-string');
+
+const {
+  translateApiCollectionToPostgresCollection,
+  translateApiProviderToPostgresProvider,
+  tableNames,
+} = require('@cumulus/db');
+
+const Collection = require('@cumulus/api/models/collections');
+const Provider = require('@cumulus/api/models/providers');
+const Pdr = require('@cumulus/api/models/pdrs');
+const {
+  createBucket,
+  recursivelyDeleteS3Bucket,
+} = require('@cumulus/aws-client/S3');
+const { dynamodbDocClient } = require('@cumulus/aws-client/services');
+const { getKnexClient, localStackConnectionEnv } = require('@cumulus/db');
+const { fakeCollectionFactory, fakeProviderFactory } = require('@cumulus/api/lib/testUtils');
+
+// eslint-disable-next-line node/no-unpublished-require
+const { migrationDir } = require('../../db-migration');
+const { migratePdrRecord, migratePdrs } = require('../dist/lambda/pdrs');
+const { RecordAlreadyMigrated } = require('../dist/lambda/errors');
+
+const migrateFakeCollectionRecord = async (record, knex) => {
+  const updatedRecord = translateApiCollectionToPostgresCollection(record);
```

Comment:
Rather than using a Dynamo fake collection and translating it to Postgres, we can just create a fake Collection PG record directly using `fakeCollectionRecordFactory` from the `db` package. It reduces the boilerplate needed for these tests and makes them a little better isolated